system/Autoloader/Autoloader.php

```diff
@@ -380,7 +383,13 @@ private function loadComposerNamespaces(ClassLoader $composer, array $composerPa
             }
         }
 
-        if (! method_exists(InstalledVersions::class, 'getAllRawData')) {
+        preg_match(
+            '/\s*"plugin-api-version": "(?\'version\'\d+\.\d+\.\d+)"/m',
+            file_get_contents(__DIR__ . '/../../composer.lock'),
+            $matches
+        );
+
+        if (version_compare($matches['version'], '2.0.14', '<')) {
```

Comment:
Why did you change the code?

Reply:
This fix phpstan error:
`Call to function method_exists() with 'Composer\InstalledVersions' and 'getAllRawData' will always evaluate to true.`
But I think, that better way will be to remove this check and check composer version during `composer install` and `composer update`...

Reply:
Okay, I got why. But your code seems to take time more.
The PHPStan error is inevitable. Because we use the latest `composer` command.
So I think the error should be suppressed by `@phpstan-ignore-next-line` or `@phpstan-ignore-line`.